app/frontend/src/features/auth/signup/Signup.tsx

```diff
@@ -64,7 +71,11 @@ function CurrentForm() {
       <>
         <BasicForm />
         <Typography variant="body1" className={classes.agreement}>
-          {SIGN_UP_AGREEMENT}
+          {SIGN_UP_AGREEMENT[0]}
+          <Link to={tosRoute} component={MuiLink}>
+            {SIGN_UP_AGREEMENT[1]}
+          </Link>
+          {SIGN_UP_AGREEMENT[2]}
```

Comment:
How come this constant is now in an array? Might be clearer to give them each a more meaningful constant name

Reply:
Because it's "Please check the <a>terms of service</a> here" and I don't want to split that up into three constants, nor do I want a function returning a react node in the constants file.

Once we implement an i18n library, this will be much easier (all the options being considered can handle cases like this).